index.bs

```diff
@@ -274,6 +275,12 @@ for plans to make {{console/timeEnd()}} and {{console/timeLog()}} formally repor
 console when a given |label| does not exist in the associated <a>timer table</a>.
 </p>
 
+<h4 id="timestamp" oldids="timestamp-label,dom-console-timestamp" method for="console">timeStamp(|label|)</h4>
```

Comment:
Maybe this should go in its own section about performance APIs? I guess this is kind of time related, but [the existing timing APIs](https://console.spec.whatwg.org/#timing) all have to do with keeping track of literal time stamps that get printed for the developer, where as `timeStamp()` seems to not interact with that timer table at all, and be more focused on interacting with a performance console.

This sounds very similar [to `console.markTimeline()`](https://github.com/whatwg/console/issues/27#issuecomment-171005496) actually. Am I correct in thinking it does not deal with actual raw times or timestamps at all?

(Also this is separate, and maybe I am dumb, but can you show me where the marked point actually shows up in the Performance panel's UI, when this API is called? I can't seem to find it when I call the API)

Reply:
> can you show me where the marked point actually shows up in the Performance panel's UI, when this API is called?

In Chrome, any call to console.timeStamp that happens during a recording is shown inside the "Timings" track.
<img width="624" alt="image" src="https://github.com/user-attachments/assets/bae39b30-1e83-4a12-be55-fbe85a1ba1ef">

Reply:
and yeah at least in Chrome, you are right about `timeStamp` being independent from the other timing APIs in `console`. There, it simply adds an entry to the data buffer of the performance panel (a trace event)

Reply:
Got it, can you confirm whether or not other browsers behave the same (or at least are as distinct from the other timing APIs as in Chrome)? That will help inform the tests we'll need for this I think.

Reply:
According to MDN, `console.timeStamp()` is solely intended for the purpose of adding data to performance traces, and from my tests with Firefox, that's exactly what happens there.

@and-oli can you confirm for Safari?

Reply:
@domfarolino Let me know if you want me to put this into a separate section on Performance APIs.

Reply:
Friendly ping.

Reply:
Sorry for the delay. Safari also uses console.timeStamp solely to add markers to the Timelines tab: [see docs](https://webkit.org/web-inspector/console-object-api/#:~:text=console.timeStamp(%5Bmessage%5D)%20adds%20a%20marker%20to%20the%20active%20timeline%20recording.)